frontend/src/components/Banner.module.scss

```diff
@@ -171,8 +170,8 @@
     display: block;
     padding: $spacing-md $spacing-lg;
     font-weight: 700;
-    background: $color-blue-50;
-    color: $color-white;
+    border: 2px solid $color-blue-50;
+    color: $color-blue-50;
```

Comment:
You'll probably want to remove the `color` from the `:focus` styles - if you now tab to this button without hovering it with your mouse, the text is white-on-white.